pkg/virt-handler/vm.go

```diff
@@ -574,8 +573,6 @@ func (d *VirtualMachineController) processVmUpdate(vm *v1.VirtualMachine) error
 	if err != nil {
 		return err
 	}
-	d.recorder.Event(vm, k8sv1.EventTypeNormal, v1.Created.String(), "VM defined.")
```

Comment:
This event is useful when retrospectively determining if virt-handler ever successfully synced the vm with virt-launcher.

```
if !vm.IsRunning() {
    d.recorder.Event(vm, k8sv1.EventTypeNormal, v1.Created.String(), "VM defined.")
}
```

We could change the name of the event from something else other than 'defined'

Reply:
Hm, we have an event which gets triggered every-time when a sync fails, I don't want to introduce on which fires everytime when it does work. Further the sync error is directly visible on the vm too.

We could reintroduce the event which directly corresponds with the libvirt "defined" event again, but right now I think about it is just an implementation detail. If defining the vm failed we will send a Warning event and report it on the vm.